### PR TITLE
fix(bash): update bash completion path [EE-6681]

### PR DESCRIPTION
**Pull request by hookenz**
[EE-6681]

[EE-6681]: https://portainer.atlassian.net/browse/EE-6681?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: kube-shell/Dockerfile [hunk 1]
```diff
@@ -2,34 +2,28 @@ ARG ALPINE=alpine:latest
 FROM ${ALPINE} as alpine
 ARG ARCH
 
+ARG KUBECTL_VERSION=v1.29.0
+ARG HELM_VERSION=v3.13.3
+
 RUN apk add -U --no-cache bash bash-completion curl jq
```
Comment (prabhat83) [inline kube-shell/Dockerfile]: how come the same changes are showing up after this got merged https://github.com/portainer/docker-images/pull/14/files

**Review by prabhat83 — APPROVED** [review]

File: kube-shell/Dockerfile [hunk 2]
```diff
@@ -2,34 +2,28 @@ ARG ALPINE=alpine:latest
...
-# RUN if [ "${ARCH}" = "amd64" ]; then ARCH=x86_64; fi && \
-#     curl -L https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9s_${K9S_VERSION}_Linux_${ARCH}.tar.gz | tar xvzf - && \
-#     chmod +x ./k9s && \
-#     mv ./k9s /usr/local/bin/k9s
-
 # Linux user permissions
 RUN echo 'shell:x:1000:1000:shell,,,:/home/shell:/bin/bash' > /etc/passwd && \
     echo 'shell:x:1000:' > /etc/group && \
     mkdir /home/shell && \
-    echo '. /etc/profile.d/bash_completion.sh' >> /home/shell/.bashrc && \
```
Comment (prabhat83) [inline kube-shell/Dockerfile:26]: this was not the issue previously. how come this is now?

Comment (hookenz) [reply kube-shell/Dockerfile:26]: We're basing on ALPINE=alpine:latest which has changed.

File: kube-shell/Dockerfile (hunk 1 above)
Comment (hookenz) [reply kube-shell/Dockerfile:9]: ?

Comment (hookenz) [reply kube-shell/Dockerfile]: woops.  Rebased.

**Pull request merged**